src/clj/api/models/user.clj

```diff
@@ -5,6 +5,8 @@
 (declare upsert-user)
 (declare get-user-by-id)
 (declare get-user-by-email)
+(declare count-user-analysis)
```

Comment:
I don't think it is important for our app but just to mention, this queries should be run inside a repeatable-read transaction isolation mode, so it doesn't happen that the search query reads some analysis, and the count one reads a different count when new analysis are inserted between the two queries and lead to broken cursors. Again just being peaky to train myself thinking about this stuff.

Reply:
good catch, there is already couple of places that could do with a transaction, I was marking them with a TODO note, so I will do so here as well. For now, as the lowest hanging fruit it's abstracted as one function call. It will be easy to wrap it in a transaction when we introduce them.